.gitignore

```diff
@@ -8,7 +8,5 @@ __pycache__/
 /hints.txt
```

Comment:
This has nothing to do with this pull request, but I just noticed that we could remove cert*.txt and hints.txt here (they are only created for inductive certificates which we won't integrate).